lib/kast.js

```diff
@@ -322,12 +372,12 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       , "_^Word__EVM-DATA"   : " ^W "
       , "_+Map__RULES"       : " +M "
       , "_andBool_"          : " AND "
-      , "_orBool__BOOL"      : " OR " 
-      , "_:__EVM-DATA"       : " : "  
+      , "_orBool__BOOL"      : " OR "
+      , "_:__EVM-DATA"       : " : "
       , "_==K_"              : " ==K "
-      , "#And"               : "\n"   
+      , "#And"               : "\n"
       , "#KSequence"         : "\n~> "
-      , "_Map_"              : "\n"   
+      , "_Map_"              : "\n"
```

Comment:
When is this used?

Reply:
e.g. `t` view - accounts are maps, storage is a map